cmd/executor/main.go

```diff
@@ -3,10 +3,14 @@ package main
 import (
 	"fmt"
 	"log"
-	"octavius/internal/executor/client"
+	healthClient "octavius/internal/executor/client/health"
```

Comment:
>Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns.

https://blog.golang.org/package-names

Reply:
alias is needed here coz the naming conflicts with daemon packages